source/overview/mattermost-cloud-overview.rst

```diff
@@ -0,0 +1,88 @@
+=========================
+Mattermost Cloud Overview
+=========================
+
+Mattermost Cloud is offered as a monthly subscription service. It is available free, indefinitely, for all workspaces that have 10 or fewer active users. Your subscription is upgraded to pay-as-you-go once you reach 11 or more active users.
+
+How Billing Works
+-----------------
+
+You are billed for active users in your workspace.  An active user: 
+
+- Has created an account and joined a workspace and shows as **Active** in **System Console > Users**.
+- Counts towards your monthly invoice for each day they are active in the billing period.
+
+Users deactivated by a System Administrator are no longer counted toward future monthly invoices.
+
+Billing periods
+~~~~~~~~~~~~~~~
+
+When you begin your Mattermost Cloud subscription, after upgrading from the free tier, your first charge is at the end of the calendar month.
+
+Billing periods end at 11:59 P.M. UTC on the final day of the month. We calculate usage for the billing period based on the number of users who were active at that time and charge the credit card stored in your profile.
+
+Adding and removing users in the middle of a billing period
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you add new users in the middle of a billing cycle, we’ll only charge for the days that the user was active. We’ll keep track of this for you, and bill you at the end of the cycle.
+
+Here’s an example:
+
+*Your organization has a Mattermost Cloud workspace and you're paying monthly by credit card at rate of $10 per user per month. You add a new user 11 days into your billing period and that user remains active for the subsequent 19 days remaining in the month - for a total of 20 days.
```

Comment:
I'm unsure how the asterisk will look when the PR is merged - currently in preview it appears clickable. 0/5 if this needs to be changed or not.

Reply:
I'll change that.